shell/renderer/api/electron_api_web_frame.cc

```diff
@@ -496,9 +496,7 @@ class WebFrameRenderer : public gin::Wrappable<WebFrameRenderer>,
 
     const auto& prefs = render_frame->GetBlinkPreferences();
 
-    if (pref_name == options::kPreloadScripts) {
-      return gin::ConvertToV8(isolate, prefs.preloads);
-    } else if (pref_name == "isWebView") {
+    if (pref_name == "isWebView") {
```

Comment:
please remove from `InternalWebPreferences` in typings/internal-ambient.d.ts as well